doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -2152,14 +1855,6 @@ Python:
     class MyError(RuntimeError):
         ROBOT_SUPPRESS_NAME = True
 
-Java:
```

Comment:
Should probably remove `Python:` line above the above example and elsewhere. It's not necessary when we only have Python examples.